core/src/test/scala/unit/kafka/log/TimeIndexTest.scala

```diff
@@ -35,7 +35,7 @@ class TimeIndexTest {
 
   @Before
   def setup() {
-    this.idx = new TimeIndex(nonExistantTempFile(), baseOffset = baseOffset, maxIndexSize = maxEntries * 12)
+    this.idx = new TimeIndex(nonExistentTempFile(), baseOffset = baseOffset, maxIndexSize = maxEntries * 12)
```

Comment:
keep same name to avoid conflict when doing next release?

Reply:
Sure, typo fix is a cosmetic change. I will rollback to the original name to avoid rebase conflicts.